tests/fixtures/00-config-sample.toml

```diff
@@ -13,6 +13,7 @@ base_url = "http://cincinnati.example.com:80/"
 allow_downgrade = true
 enabled = false
 strategy = "fleet_lock"
+use_oci = false
```

Comment:
I think in our discussions we came to the conclusion that we didn't need this knob. i.e. inspecting the system state to see it was deployed via container or not would suffice and then we'd change behavior based on that.

Reply:
So you would have to manually rebase, then zincati follow ? 
I thought you'd use the knob to migrate